website/docs/getting-started/setup/upgrade-to-business-edition/README.md

```diff
@@ -1,29 +1,42 @@
+---
+description: Follow the steps to upgrade the Appsmith installation to a Business Edition.
+---
 # Upgrade to Business Edition
 
-Follow the below guides to **upgrade** your **Community Edition (CE)** to a **Business Edition (BE)** based on the type of your **Appsmith Installation:**
+Follow the steps below to **upgrade** your **Community Edition (CE)** to a **Business Edition (BE)**. Choose the type of  **Appsmith Installation** to get started.
 
-:::note
-You'll need your  [**Business Edition License Key**](https://www.appsmith.com/pricing)  to complete the upgrade.
+:::info License Key
+You would need a **Business Edition License Key** to complete the upgrade. Learn more on [**Business Edition & pricing**](https://www.appsmith.com/pricing) options.
 :::
 
-
-* [Docker](docker.md)
-
-
-:::note
-For **non-docker Appsmith installations**, please get in touch with us via [discord](https://discord.com/invite/rBTTVJp) or [email](mailto:support@appsmith.com). We will help you upgrade your Appsmith instance.
-:::
-
-## License Renewal
-
-The Appsmith server shuts down if your Business license key is expired. You will see an invalid license key message like below in the logs:
-
-```
+<div class="containerBorder">
+    <div class="containerGrid">
+        <div class="columnGrid column-one" align="center">
+            <div class="containerCol">
+                <a href="docker/">
+                <img class="containerImage" src="/img/docker-logo.png" alt="Docker-logo"/>
+                </a> 
+            </div> 
+            <b><a href="docker/">Docker</a></b>
+        </div>
+        <div class="columnGrid column-two" align="center">
+       </div>
+        <div class="columnGrid column-three" align="center">
+        </div>
+    </div>
+</div>
+
+## License renewal
```

Comment:
minor nit pick. For some reason I feel this heading should be "Renew License" . Mostly because throughout the docs we start headings with a verb.

Reply:
Hmm.

Reply:
Is this link been shared with users? I will have to add a redirect URL rule if we change the heading.

Reply:
It might have been shared

Reply:
I can add a redirect URL rule, but with our experience earlier with old and new URLs should we check with Tanmay if we can add redirect URLs?

Reply:
Yes, let us confirm with Tanmay first. I'll check with him. Thanks for pointing that out.

Reply:
Checked with Tanmay. He said as it is already there, leave it as is in case these links have been shared to avoid redirect URLs. But going forward, we should make sure headings are consistent throughout.